### ISSUE TITLE
error building rest module on Jenkins

### ISSUE THREAD

**Issue by zregvart** [syndesisio/syndesis#458]
We have this error when building on Jenkins (see [#456 build](https://jenkins-syndesis-ci.b6ff.rh-idev.openshiftapps.com/blue/organizations/jenkins/syndesis/detail/PR-456/1/pipeline))

Command:
```
mvn -B -U clean install fabric8:build -Pci
```
Error:
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-checkstyle-plugin:2.17:check (default) on project syndesis-rest-parent: Failed during checkstyle execution: Unable to find configuration file at location: ${syndesis.basedir}/checkstyle.xml: Could not find resource '${syndesis.basedir}/checkstyle.xml'. -> [Help 1]
```

**Comment by rhuss** [comment]
@zregvart  My fault.

I removed the syndesis.basedir as I thought it was only used for the license check (and I inlined now the license text, removing header.txt)

Going to fix this asap.

**Comment by zregvart** [comment]
I wonder if all of this will be fixed in #411

**Comment by zregvart** [comment]
This is now fixed, we still have other issues on Jenkins, but not this one.
